thetis/rungekutta.py

```diff
@@ -424,8 +424,8 @@ class ESDIRKMidpointAbstract(AbstractRKScheme):
 
 class ESDIRKTrapezoidAbstract(AbstractRKScheme):
     a = [[0.0, 0.0],
-         [0.5, 0.5]]
-    b = [0.5, 0.5]
+         [0.499, 0.501]]
+    b = [0.499, 0.501]
```

Comment:
Why?

Reply:
In the `baroclinic_channel` large-scale test a numerical mode develops in the elevation field. It's really subtle, takes more than a year to show up. My guess is that it's related to the weak instability that Crank-Nicolson scheme manifests. This is just to nudge the scheme to be a bit more diffusive - seems to solve the issue. I'll double check that it's still needed, there have been some changes in the time integrator.

Reply:
OK, the weak instability still seems to occur in the `baroclinic_channel` test. However this commit does affect the convergence rate of the free surface solver, so I'm going to drop it for now. I hope I can find another workaround for that specific test.